### PR TITLE
Lecture 06-b

### PR DESCRIPTION
**Pull request by ql819**
In this note, we introduce how to apply ConvNets on graphs, like social networks, knowledge graphs. In addition, we cover how recurrent neural networks work and how to do backpropagation through time. We also talk about the issue of the simple recurrent net including exploding gradient and vanishing gradient. Finally, we introduce some approach to solve these issues like gradient clipping, the RNN capable of learning long-term dependencies.

-- Second part of Lecture 6 (11 May 2019) by Yuqiong Li and Qintai Liu, edited by Aaron L Zweig

### CODE REVIEW

**Comment by Atcold** [comment]
PR summary and date are missing.
Plz, edit the first message, similarly to how it's done in #1.

**Comment by Atcold** [comment]
Also, what about #26?

**Comment by ql819** [comment]
> Also, what about #26?

Just ignore it. Sorry for the confusion.

**Comment by Atcold** [comment]
Thanks!

**Comment by Atcold** [comment]
Plz, fix.

Screenshot 2019-03-19 17 20 26

**Comment by ql819** [comment]
> Plz, fix.
> 
> Screenshot 2019-03-19 17 20 26

Hi, I just opened the latest version of lecture note, the content was compiled appropriately. I didn't see the error you showed in the screenshot

**Comment by Atcold** [comment]
Next to the *Recompile* button there's a orange label with a white number.
Plz, fix the warnings listed there. Thanks.

Screenshot 2019-03-20 14 45 12

**Comment by ql819** [comment]
> Next to the _Recompile_ button there's a orange label with a white number.
> Plz, fix the warnings listed there. Thanks.
> 
> Screenshot 2019-03-20 14 45 12

I just sent a pull request(#29) which fix the issue.

**Comment by Atcold** [comment]
I saw that. Please, rebase your commit on top of `master`.
You've been editing end pushing old stuff (see *Files changed* tab, on #29).

**Comment by ql819** [comment]
> I saw that. Please, rebase your commit on top of `master`.
> You've been editing end pushing old stuff (see _Files changed_ tab, on #29).

Sorry for the inconvenience. I just resent the pull request(#30) and please close request(#29)

**Comment by Atcold** [comment]
You didn't need to send another PR. You could force push.
